src/portable/sony/cxd56/dcd_cxd56.c

```diff
@@ -329,6 +329,7 @@ bool dcd_edpt_xfer(uint8_t rhport, uint8_t ep_addr, uint8_t *buffer, uint16_t to
   {
     if (total_bytes == 0)
     {
+      usbdcd_driver.setup_processed = true;
```

Comment:
if you want to detect status packet with this logic, it can mis-inteprete other event. For example when receiving 64 bytes of a descriptor, to end the data transfer, ZLP is sent in the DATA state on the IN endpoint. Afterwards there is another ZLP on OUT endpoint for status.